src/hooks/useBuyout/types.ts

```diff
@@ -0,0 +1,13 @@
+export interface OrmlTraitsAssetRegistryAssetMetadata {
```

Comment:
Nice. I can also show you how to add types to our [pendulum.js](https://github.com/pendulum-chain/pendulum.js) types package in the near future. Let's do this once we fixed the asset registry data, ie. probably next week.